app/components/mandatenbeheer/mandaat-bestuursorganen-selector.js

```diff
@@ -0,0 +1,37 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+import { inject as service } from '@ember/service';
+
+export default class MandaatBestuursorganenSelector extends Component {
+  @service store;
+
+  @tracked mandaat = null;
+  @tracked mandaten = null;
```

Comment:
mandaat - mandaten, might be a bit confusing, might be a good idea to indicate the difference with selected or with options?